### PR TITLE
Add installation instructions for modifiers

### PR DESCRIPTION
**Pull request by jrjohnson**
Referencing the addon probably isn't sufficient for new users and was
confusing for existing Emberistas as well since it sometimes isn't clear what is
in the default blueprint or included in another way. Adding installation
instructions clears that up.

### CODE REVIEW

**Comment by jrjohnson** [comment]
@jenweber I missed that entirely, thanks. I'm going to advocate putting it in both places for the same reason that I missed this when reading the docs last night. I was scanning headings, found what I wanted and needed instructions at that point.  I'm happy to remove the second set of instructions though if they seem too duplicative.

File: guides/release/components/glimmer-components-dom.md [hunk 1]
```diff
@@ -288,7 +288,14 @@ The modifier that we're going to build will allow us to say:
 
 Pretty nice, right?
 
-To accomplish that, we'll create a modifier in `app/modifiers/autofocus.js` after first installing [`ember-modifier`](https://github.com/ember-modifier/ember-modifier):
+To accomplish that, we'll create a modifier in `app/modifiers/autofocus.js` first install [`ember-modifier`](https://github.com/ember-modifier/ember-modifier) and generate an `autofucus` modifier for your app:
```
Comment (jenweber) [inline guides/release/components/glimmer-components-dom.md]:
```suggestion
To accomplish that, we'll create a modifier in `app/modifiers/autofocus.js`. First, install [`ember-modifier`](https://github.com/ember-modifier/ember-modifier) and then generate an `autofocus` modifier for your app:
```

**Comment by jrjohnson** [comment]
Thanks @jenweber. I merged your changes, let me know if you'd rather I rebased everything into a single commit.

**Comment by rwjblue** [comment]
I've targeted `master` branch, and merged master into this branch (to try to get CI to pass).

**Pull request merged**